variables.tf

```diff
@@ -103,7 +68,18 @@ variable "agent_aws_secret_access_key" {
 }
 
 variable "castai_components_labels" {
-  type = map
+  type        = map
```

Comment:
nitpick: Why ąnot object?
```
  type     = map(object({
  disk_cpu_ratio = any
  subnets = any
  })
```

Using an object will serve as documentation of what fields are available to use